QUEST_LV_0100.tsv

```diff
@@ -585,58 +585,58 @@ QUEST_LV_0100_20150317_000584	$Miners' Village Mayor
-QUEST_LV_0100_20150317_000633	Evil energies are pervading all over the Mausoleum.{nl}We can no longer remain pervasion.
-QUEST_LV_0100_20150317_000634	It will no longer be polluted.
+QUEST_LV_0100_20150317_000622	$The demons have cut off the source of magic.{nl}We need you to gather magical power from the nearby protectors.
+QUEST_LV_0100_20150317_000623	$Good.{nl}Now, there is a lack of magical power.
+QUEST_LV_0100_20150317_000624	$I will open the way to the problem.{nl}Restore our magical power by reigniting the stone lantern of Mausoleum.
+QUEST_LV_0100_20150317_000625	$When the suppressed power flows yet again, the protectors will regain their reason.{nl}But it would take a long time.
+QUEST_LV_0100_20150317_000626	$Abnormal protectors are attacking the magical container.{nl}Protect the magical container until the suppressed power flows again.
+QUEST_LV_0100_20150317_000627	$The magical container is like the heart of Mausoleum.{nl}We can never let it be destroyed.
+QUEST_LV_0100_20150317_000628	$Order will surely return to this place.{nl}But it will take a long time.
+QUEST_LV_0100_20150317_000629	$Evil energies have condensed everywhere in the Mausoleum with the current of suppressed power.{nl}Find and remove the corrupted lumps.
+QUEST_LV_0100_20150317_000630	$The Mausoleum was designed perfectly.{nl}But the demons' plan seems beyond perfect.
+QUEST_LV_0100_20150317_000631	$Our protectors are even under their control, and even do their mission.{nl}This is our failure.
+QUEST_LV_0100_20150317_000632	$Evil energy has corrupted even the stone lanterns which were the origins of magical power.{nl}Destroy the corrupted stone lanterns to stop the spread of corruption.
+QUEST_LV_0100_20150317_000633	$Evil energy is pervading all over the Mausoleum.{nl}We can no longer remain idle.
+QUEST_LV_0100_20150317_000634	It is no longer corrupted.
 QUEST_LV_0100_20150317_000635	Mausoleum Document
-QUEST_LV_0100_20150317_000636	Wheelen's mark give us power to face against those vicous beings.
-QUEST_LV_0100_20150317_000637	Venucelos is very brave and loyal.{nl}But if his loyalty faces towards vicious beings, he will be worthless.
-QUEST_LV_0100_20150317_000638	Zachariel was proud to make huge protectors.{nl}And he put secret letters inside them so that only the one who destroy the protectors could get them.
-QUEST_LV_0100_20150317_000639	Vicious beings can't reach near the revelation of the Goddess.
+QUEST_LV_0100_20150317_000636	Wheelen's mark give us power to face against the forces of evil.
+QUEST_LV_0100_20150317_000637	$Venucelos are very brave and loyal.{nl}But, if their loyalty are the enemies', they are worthless.
+QUEST_LV_0100_20150317_000638	$Zachariel was proud to make huge protectors.{nl}He even put secret treasures inside them so that only the one who destroy these protectors could get them.
```

Comment:
Zachariel -> The Great King Zachariel
huge -> gigantic
protectors -> guardians (2x)